### PR TITLE
Linting

### PR DESCRIPTION
**Pull request by jcpunk**
This patch set should clean up a number of linter warnings.

### CODE REVIEW

File: manifests/fsck.pp [hunk 1]
```diff
@@ -5,43 +5,40 @@
...
 
   if $_usesystemd {
-
-    systemd::unit_file{'cvmfs-fsck.timer':
+    systemd::unit_file { 'cvmfs-fsck.timer':
       ensure  => present,
       content => epp('cvmfs/fsck/cvmfs-fsck.timer.epp',
-      {
-        'onreboot' => $onreboot,
+        {
```
Comment (traylenator) [inline manifests/fsck.pp:20]: Looks wrong, probably need to push that last bracket to it's own line.

File: manifests/fsck.pp [hunk 2]
```diff
@@ -5,43 +5,40 @@
...
+    systemd::unit_file { 'cvmfs-fsck.service':
       ensure  => present,
       content => epp('cvmfs/fsck/cvmfs-fsck.service.epp',
-      {
-        'cache_base' => $cvmfs_cache_base,
-        'options'    => $options,
+        {
+          'cache_base' => $cvmfs_cache_base,
+          'options'    => $options,
       }),
```
Comment (traylenator) [inline manifests/fsck.pp:32]: Looks wrong, probably need to push that last bracket to it's own line.

File: manifests/fsck.pp [hunk 3]
```diff
@@ -5,43 +5,40 @@
...
     }
-    systemd::tmpfile{'cvmfs-quarantaine.conf':
+    systemd::tmpfile { 'cvmfs-quarantaine.conf':
       content => epp('cvmfs/fsck/cvmfs-quarantaine.conf.epp',
-      {
-        'cache_base' => $cvmfs_cache_base,
+        {
+          'cache_base' => $cvmfs_cache_base,
       }),
```
Comment (traylenator) [inline manifests/fsck.pp:38]: Push last bracket to it's own line.

**Review by traylenator — COMMENTED** [review]
Also happy to add the stronger linting but could the Gemfile be updated to add the extra lint checks else it won't stay that way, thanks.

**Comment by jcpunk** [comment]
> could the Gemfile be updated to add the extra lint checks

I'll confess I've no idea how to do that....

**Comment by traylenator** [comment]
Okay, I'll add an extra MR after this one, looks fine but I'll look again with fresh eyes on Monday.

**Comment by jcpunk** [comment]
Just checking back in.

**Comment by traylenator** [comment]
I've moved the module now to vox pup space where the linting is more up todate.

Next release will have this resolved.

**Comment by traylenator** [comment]
Next release will have this resolved.

Thanks for the patch, so many other lint fixes hard to take this patch.

**Pull request closed without merging**